packages/ckeditor5-table/tests/ui/colorinputview.js

```diff
@@ -218,6 +218,23 @@ describe( 'ColorInputView', () => {
 				expect( inputView.value ).to.equal( 'bar' );
 			} );
 
+			it( `when the color input value is set to one of defined colors, but with few additional white spaces,
+			should use its label as the text input value`, () => {
```

Comment:
I'm not sure if I like that notation. Could you try to make it fit?

Reply:
I hope that's enough https://github.com/ckeditor/ckeditor5/pull/6801/commits/fc82d5cec73a5d1456fc3b742ee05423d0fadc09
I don't think, I can rephrase the text to make it fit.

This is the styling we agreed on with @Reinmar in the previous review. We both agreed it's not perfect, but acceptable.
If you have any better ideas, please let me know.

Reply:
Just to be precise :) https://github.com/ckeditor/ckeditor5/pull/6801/files#r425600735

The test description is awfully long for me. It's my proposition to shorten this, but I'm not insisting on that form. The fc82d5c is a bit worse IMO here. The trick with template string was OK.

Reply:
OK I've read rest of the test. As in other comment - leave it as is.